compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/test/unroll_and_distribute_workgroup.mlir

```diff
@@ -190,11 +190,55 @@ module {
 
 // -----
 
+// Check for unrolling a parallel loop, with both cores and dma ops. The dma op
+// does't depend on one of the induction variables and can be hoisted, resulting
+// in a single dma op in the output instead of two. However, in this test, the
+// DMA operation does depend on an affine apply operation within the `scf.for`
+// operation's scope and checks whether both the affine apply and the DMA can
+// be hoisted.
+//
+// CHECK-LABEL: @hoist_dma_and_affine_single_loop
+// CHECK:       scf.forall
+// CHECK:         amdaie.workgroup
+// CHECK:           amdaie.dma_cpy_nd
+// CHECK-NOT:       amdaie.dma_cpy_nd
```

Comment:
This is quite confusing. I'd suggest to expand the check contents to include affine.apply, amdaie.core, amdaie.tile, etc...

Reply:
Done, but there is no affine.apply anymore in the end state